pkg/logs/launchers/container/tailerfactory/file_test.go

```diff
@@ -317,7 +317,7 @@ func TestMakeK8sSource(t *testing.T) {
 			require.Equal(t, wildcard, child.Config.Path)
 			require.Equal(t, "src", child.Config.Source)
 			require.Equal(t, "svc", child.Config.Service)
-			require.Equal(t, logConfig.TagsField{"tag!"}, child.Config.Tags)
+			require.Equal(t, Config.TagsField{"tag!"}, child.Config.Tags)
```

Comment:
FYI I also had issues trying to get the equality check to work like this. So I used 
`assert.EqualValues` instead: https://github.com/DataDog/datadog-agent/pull/34065/files#diff-443361ce7f93f980a63c97a327dc787aafd89605b9b1e4164c5e9257ca5b8ce8R30